src/main/java/org/apache/commons/mail/util/AttachmentDataSource.java

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
Looks like a copy of https://github.com/apache/cxf/blob/main/rt/frontend/jaxrs/src/main/java/org/apache/cxf/jaxrs/ext/multipart/InputStreamDataSource.java ?

Reply:
Oh! It's almost the same.